public/css/style.css

```diff
@@ -357,6 +357,10 @@ label {
   margin: 0;
 }
 
+#notificationIcon {
```

Comment:
@lousando Get with the program, man! JK. There's no written-down convention. It's all in my head.